src/main/java/biz/paluch/logging/gelf/Values.java

```diff
@@ -13,35 +13,35 @@
  */
 public class Values {
 
-    private Map<String, Object> values = new HashMap<>();
+    private Map<String, Object> keyValueMap = new HashMap<>();
```

Comment:
Not sure this is a helpful change.

Reply:
I know what you mean ... naming a field with the same name of the class was the SonarLint issue. I think no change is better than just making a tool happy.